rpmlint/checks/LSBCheck.py

```diff
@@ -18,15 +9,50 @@
 
```

Comment:
The regexps could be class variable maybe, but regardless the a-zA-Z0-9... should be \w+ see the https://docs.python.org/3/howto/regex.html